helm/charts/services/amplication-server/templates/job.yaml

```diff
@@ -3,6 +3,7 @@ kind: Job
 metadata:
   name: "{{ .Values.name }}-migrate"
 spec:
+  ttlSecondsAfterFinished: 5
```

Comment:
This configuration is not solving that issue?

Reply:
it should